css/custom.css

```diff
@@ -1068,41 +1068,41 @@ pre.highlight>code {
 }
 
 .arrow-down {
-  width: 0; 
-  height: 0; 
+  width: 0;
+  height: 0;
   border-left: 3px solid transparent;
   border-right: 3px solid transparent;
   border-top: 3px solid #393939;
 }
 
 @media only screen and (max-width: 75rem) {
-    
+
```

Comment:
While I absolutely agree with removing trailing spaces, it's apparently not something greatly appreciated (especially in this case as so many unrelated lines are updated).